src/modules/managers/managers.module.ts

```diff
@@ -2,9 +2,11 @@ import { Module } from '@nestjs/common';
 import { ManagersService } from './managers.service';
 import { ManagersController } from './managers.controller';
 import { ManagersRepository } from './managers.repository';
+import { UsersModule } from '../users/users.module';
 
 @Module({
+  imports: [UsersModule],
```

Comment:
I didn´t realize the use of this module inside managers, I need to double check this information.